velodrome/grafana-stack/README.md

```diff
@@ -1,7 +1,29 @@
+Overview
+========
+
 The goal of this directory is to set-up the following monitoring stack:
 - InfluxDB as the Time-series Database
 - Grafana as the front-end/display
-- Nginx is use as a proxy (mostly to fix CORS issue)
+- Prometheus as a different flavor of time-series database
+- Nginx is used as a proxy (mostly to fix CORS issue)
+
+Testing locally
+===============
+
+You can set-up your own local grafana-stack easily. In order to do that, you
+probably don't need to set-up passwords or even the `nginx` proxy.
+
+You can run:
+
+```
+docker run -d -p 8083:8083 -p 8086:8086 tutum/influxdb:0.13
+docker run -i -p 3000:3000 grafana/grafana
+docker run -p 9090:9090 -v prom/prometheus
+```
+
+You will probably need to set-up the datasources/users into Grafana and InfluxDB as described
```

Comment:
You don't like "probably" ? I agree that it sounds like the docs doesn't even know what's going on :-)